katalog/nginx/deployment.yml

```diff
@@ -14,7 +14,7 @@ spec:
       serviceAccountName: nginx-ingress-serviceaccount
       containers:
         - name: nginx-ingress-controller
-          image: quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.20.0
```

Comment:
the update is breaking the deployment, you can verify it by deploying it to your local machine